### DISCUSSION TITLE
Using Web3Auth with nextjs 13 results in error

### DISCUSSION THREAD

**Discussion by yashovardhan** [Web3Auth/Web3Auth#930] [Help]
Related to this discussion on stack overflow: https://stackoverflow.com/questions/74222190/importing-web3-to-next-js-results-in-error

The web3.js version being used in Web3Auth is `1.8.0` however, it is incompatible as mentioned in the stack overflow discussion. Can you help me roll it back to a previous version?

> Asked on a call discussion with a developer

**Comment by shahbaz17** [comment]
Simply upgrade your node version to `v18`

**Reply by yashovardhan** [reply]
As mentioned in [this issue](https://github.com/web3/web3.js/issues/5601) on web3.js, you can do the following combinations

web3.js v1.8.0 + Next.js v13.0.2 + Node.js v18 → all good!
web3.js v1.8.0 + Next.js v13.0.2 + Node.js v16 → broken with reported error
web3.js v1.7.4 + Next.js v13.0.2 + Node.js v16/v18 → all good!

Use Node v18 ideally. If you want to use Node v16, you can enforce in your package json to use web3.js v1.7.4
